File: Data Structures & Algorithms/two-integer-sum-ii/submission-1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        vector<int> res;
        int n=numbers.size();
        for(int i=0;i<n;i++)
        {
            for(int j=i+1;j<n;j++)
            {
                if(numbers[i]+numbers[j]==target)
                {
                    res.push_back(numbers[i]);
                    res.push_back(numbers[j]);
                    return {i+1,j+1};
                }
            }
        }
        return {};
    }
};

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-1.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        int[] res=new int[nums.length];
        for(int i=0;i<nums.length;i++){
            int prdt=1;
            for(int j=0;j<nums.length;j++){
                if(j!=i)
                prdt*=nums[j];

            }
            res[i]=prdt;
        }
        return res;

        
    }
}  
